f/molecule.json

```diff
@@ -206,6 +206,13 @@
           },
           "title": "Volumes",
           "type": "array"
+        },
+        "env": {
```

Comment:
As far as I'm correct, this adds `env` for all platforms, but that property is specific to docker/podman containers. See https://github.com/ansible-community/schemas/issues/83#issuecomment-887645260 for details. This property should be added conditionally.

Reply:
Ah, so should we be separating out docker-specific tags into an optional schema?

Looking at the [MoleculeDriverModel options](https://github.com/ansible/schemas/blob/ea6b7237c376e483d9c4f4fd439a0da9fdf3a120/f/molecule.json#L51-L63), it would make sense to have separate options for each of those. I'm super new to this space, is it possible to derive options based on a field somewhere else in the template?

Similarly, should options like [dockerfile](https://github.com/ansible/schemas/blob/ea6b7237c376e483d9c4f4fd439a0da9fdf3a120/f/molecule.json#L116-L119) only be allowed if using `podman` or `docker` ?

Reply:
Yes, docker-specific options **should** be put separate. Not in a separate schema, but with JSON schema conditionals. For details see here: https://json-schema.org/understanding-json-schema/reference/conditionals.html

And yes, there are already docker/podman specific options in the general Platform Module, but in the best case that would change in the future too. See also #83.

Reply:
We will have to deal with this at a later time, when we would make a new schema. For now, that is ok.